Sources/Components/Colors.swift

```diff
@@ -0,0 +1,49 @@
+import UIKit
+
+public extension UIColor {
+
+	static let gray800 = UIColor(0x101418)
+	static let gray700 = UIColor(0x202122)
+	static let gray675 = UIColor(0x27292D)
+	static let gray650 = UIColor(0x2E3136)
+	static let gray600 = UIColor(0x54595D)
+	static let gray500 = UIColor(0x72777D)
+	static let gray450 = UIColor(0x8E8E93)
```

Comment:
I don't think this color is used anywhere.

Reply:
I removed it, we can add it back if it is ever necessary

Reply:
I don't see the change, might need a push?